R/compute_abm_metrics.R

```diff
@@ -43,7 +43,8 @@ compute_abm_metrics <- function(abm_result) {
       transmissions = replace_na(transmissions, 0),
       found = !is.na(case_found_by),
       traced = found & case_found_by == "contact_tracing",
-      screened = found & case_found_by == "screening",
+      screened = found & case_found_by == "workplace_screening",
+      symptomatic = found & case_found_by == "passive_surveillance",
```

Comment:
for avoiding further future confusion, could you please rename all instances of `"passive_surveillance"` to `"symptom_surveillance"`, since passive might also be considered to include workplace screening

Reply:
done.